core/controllers/base.py

```diff
@@ -53,7 +53,11 @@
 
 
 def _clear_login_cookies(response_headers):
-    """Clears login cookies from the given response headers."""
+    """Clears login cookies from the given response headers.
```

Comment:
@lkbhitesh07 This change looks similar to the changes in #7751. 

I think you have created a new branch off of your feature branch. Have you followed the [wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change) correctly?

Reply:
I found two issues with your PR:
1. The wiki page asks contributors to complete the build process before starting working on any issue and I don't think you have locally started oppia server once.
2. The Wiki page asks the contributor to initially checkout to the develop branch and then create a new branch off of it (stated [here in step 2 of the wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)). In your case, you have create this branch from the feature branch and that's why the changes in the previous branch still exist in your new branch. 

I would suggest you to thoroughly read and follow the [steps provided in the wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change) that would help you avoid this kinda issue(We all face such issue in our initial contribution and following the guidelines provided in the wiki page helps everyone!)

Reply:
@DubeySandeep I've tried installing oppia many time to my windows, but every time i got some issues, actually i'm having many trouble using windows, so i was thinking to change to linux environment. So, i'll be able to make these requested changes tomorrow.
Thankyou